skipper.go

```diff
@@ -1357,6 +1363,14 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		)
 	}
 
+	if bytes, err := getCgroupV1MemoryBytes(); err != nil {
+		o.CustomFilters = append(o.CustomFilters, script.NewLuaScript())
+	} else {
+		if int(bytes) > 2*(1<<30) {
+			o.CustomFilters = append(o.CustomFilters, script.WithPoolSize(10000))
+		}
+	}
```

Comment:
Config option might be a better solution. Skipper operator knows its memory size upfront and we may provide guidance on setting max pool size based on available memory and benchmark numbers.

Reply:
yes, this was just the quick hack :)